x-pack/plugins/case/server/connectors/index.ts

```diff
@@ -41,3 +49,19 @@ export const externalServiceFormatters: ExternalServiceFormatterMapper = {
   '.jira': jiraExternalServiceFormatter,
   '.resilient': resilientExternalServiceFormatter,
 };
+
+export const isCommentGeneratedAlert = (
+  comment: CommentSchemaType | CommentRequest
+): comment is ContextTypeGeneratedAlertType => {
+  return (
+    comment.type === CommentType.generatedAlert &&
+    'alerts' in comment &&
+    comment.alerts !== undefined
+  );
+};
+
+export const isCommentAlert = (
```

Comment:
nice 👍🏾, this was the idea for all the other types of checks as well that I mentioned earlier

Reply:
Ah got it.

Reply:
nit: I see this is unused. Maybe we can remove it.